mozilla_version/maven.py

```diff
@@ -0,0 +1,56 @@
+"""Defines characteristics of a Maven version at Mozilla."""
+
+import attr
+import re
+
+from mozilla_version.version import BaseVersion
+
+
+@attr.s(frozen=True, cmp=False, hash=True)
+class MavenVersion(BaseVersion):
+    """Class that validates and handles Maven version numbers.
+
+    At Mozilla, Maven packages are used in projects like "GeckoView" or Android-Components.
```

Comment:
Tiny nit: for consistency, maybe we should either include both the projects in double-quotes, or none?

Reply:
Done